src/entercommand.js

```diff
@@ -4,22 +4,29 @@
  */
 
 import Command from '../core/command/command.js';
-import Element from '../engine/model/element.js';
-import LivePosition from '../engine/model/liveposition.js';
-import Position from '../engine/model/position.js';
 
 /**
- * The Enter command. It is used by the {@link enter.Enter Enter feature} to handle the <kbd>Enter</kbd> key.
+ * Enter command. It is used by the {@link enter.Enter Enter feature} to handle the <kbd>Enter</kbd> key.
  *
  * @member enter
  * @extends core.command.Command
  */
 export default class EnterCommand extends Command {
+	/**
+	 * Executes command.
+	 *
+	 * @protected
+	 * @returns {Object} data Data object, available in {@link core.Command#event:afterExecute}
+	 * @returns {engine.model.Batch} data.batch Batch created and used by the command.
+	 */
 	_doExecute() {
 		const doc = this.editor.document;
+		const batch = doc.batch();
 
 		doc.enqueueChanges( () => {
-			enterBlock( doc.batch(), doc.selection, { defaultBlockName: 'paragraph' } );
+			enterBlock( batch, doc.selection );
+
+			this.fire( 'afterExecute', { batch } );
```

Comment:
This will be automated but doing it now would will take us too much time and it's better to close this part of code and then rework all the commands.